### PR TITLE
fix: add builder journey

### PR DESCRIPTION
**Pull request by schuettc**
**Issue #:**

**Description of changes:**

Add builder journey doc to guides

**Testing:**
Doc changes only

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*

**Checklist:**

1. Have you successfully run `npm run build:release` locally?

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: guides/20_Builders_Journey.md [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+- [Serverless Demo](https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/serverless)
+- [Developer Guide](https://docs.aws.amazon.com/chime/latest/dg/using-the-messaging-sdk.html)
+
+### 2.2 Quotas / Limits
+
+AWS has service quotas (previously call service limits) to help prevent accidental provisioning of more resources than needed and to help ensure availability of AWS resources.
+
+For each AWS account there are hard limits that cannot be adjusted and soft limits which can be adjusted:
+
+**Hard limits:**
```
Comment (devalevenkatesh) [inline guides/20_Builders_Journey.md:117]: This is good to have here, but, we may have to track to update this when the limits are changed in AWS docs. So could you please add a link to AWS docs here.

File: guides/20_Builders_Journey.md [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+
+According to the Meriam Webster Dictionary a Proof of Concept is "something that demonstrates the feasibility of a concept". An Amazon Chime SDK POC is usually designed to help validate that core goals can be met using the Amazon Chime SDK. It is not generally a full-blown application and is not usually deployed to the masses but is tested and verified by a small subset of folks, often just within IT.
+
+###
+
+### 3.1 What is a POC Readiness Review?
+
+It is important that you what your goals are. You document what they are trying to accomplish in the POC in the POC Readiness Review. In this review, you and AWS review the timelines, success criteria, and expectations.
+
+### 3.2 POC Readiness Review
```
Comment (devalevenkatesh) [inline guides/20_Builders_Journey.md:209]: Should we also tell builders how they can create this ticket if needed or what would be the process for engaging with the SAs?

**Review by devalevenkatesh — DISMISSED** [review]

File: docs/classes/audioprofile.html [hunk 3]
```diff
@@ -152,7 +152,7 @@ <h3>audio<wbr>Bitrate<wbr>Bps</h3>
 					<div class="tsd-signature tsd-kind-icon">audio<wbr>Bitrate<wbr>Bps<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">number</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/aws/amazon-chime-sdk-js/blob/main/src/audioprofile/AudioProfile.ts#L10">src/audioprofile/AudioProfile.ts:10</a></li>
+							<li>Defined in <a href="https://github.com/schuettc/amazon-chime-sdk-js/blob/main/src/audioprofile/AudioProfile.ts#L10">src/audioprofile/AudioProfile.ts:10</a></li>
```
Comment (michhyun1) [inline docs/classes/audioprofile.html:155]: Not sure why your alias is here

**Comment by michhyun1** [comment]
Please fix the alias being in the generated docs

**Pull request closed without merging**